articles/getting-started-with-selenium-automation-testing/index.md

```diff
@@ -0,0 +1,191 @@
+
+A new window will pop-up requiring you to fill the name of the package. You can name it **newpackage**. Click on 'Finish'.
+
+**Step 5:** Right-click on the **newpackage** to generate a Java class. Select *New* and click on **Class**. 
+
+![Step 5-1](/engineering-education/getting-started-with-selenium-automation-testing/step-5-1.jpg)
+
+This will generate a pop-up window that requires you to fill in the name of the class. Click on *Finish* when done. After creating the class, the Eclipse interface will appear as follows.
+
+![Step 5-2](/engineering-education/getting-started-with-selenium-automation-testing/step-5-2.png)
+
+**Step 6:** The next step involves adding Selenium Jars to the newly created project. Right-click on the folder **newproject** and click on *properties*. 
+
+![Step 6-1](/engineering-education/getting-started-with-selenium-automation-testing/step-6-1.jpg)
+
+This will generate a pop-up window. On this window, click on *Java Build Path* and select *Libraries*. 
+
+![Step 6-2](/engineering-education/getting-started-with-selenium-automation-testing/step-6-2.jpg)
+
+**Step 7:** Click on *Add External JARS* to add Selenium JARS. 
+
+![Step 7](/engineering-education/getting-started-with-selenium-automation-testing/step-7.jpg)
+
+Step 8: Search for the folder containing the downloaded Selenium Client Driver. Click on the folder *libs* in the downloaded Selenium folder. 
+
+![Step 8-1](/engineering-education/getting-started-with-selenium-automation-testing/step-8-1.jpg)
+
+Highlight all the jar files and click *open*.
+Highlight the two jar files outside the lib folder and click *open*. 
+
+![Step 8-2](/engineering-education/getting-started-with-selenium-automation-testing/step-8-2.png)
+
+Step 9: Click on *Apply and Close*. 
+
+![Step 9](/engineering-education/getting-started-with-selenium-automation-testing/step-9.jpg)
+
+Step 10: Check whether the following two library folders appear on your Eclipse dashboard. If this is the case, then you have successfully configured Selenium WebDriver with Eclipse.
+
+![Step 10](/engineering-education/getting-started-with-selenium-automation-testing/step-10.jpg)
+
+### Running The First Test in Selenium WebDriver
+
+Let’s test whether a specific URL such as `google.com` functions well. We will conduct this test using the Chrome browser. Every browser used in Selenium testing has a unique browser driver. In our case, we need to download the Chrome driver. This can be downloaded from [here]( https://sites.google.com/a/chromium.org/chromedriver/). Select the Chrome driver that corresponds to your operating system.
+
+![Chrome Driver](/engineering-education/getting-started-with-selenium-automation-testing/chrome-driver.png)
+
+Before launching the browser, we need to set the path of the downloaded executable file. The following is an example of how the property can be set.
+
+```java
+System.setProperty("webdriver.chrome.driver", "D:\\ChromeDriver\\chromedriver.exe");
+```

```

Comment:
@mbaabuones Before starting with this step, please instruct the reader to create a class file and add import statements for necessary classes.

After that, instruct the users to write all of this in the main function.

Reply:
@zolomohan Done. Thank you for these meaningful insights.

Reply:
It's better to add code snippets every time while explaining. I did it this time, but keep this in mind for your next article.